### ISSUE TITLE
[Feature] Hide blocked devices

### ISSUE THREAD

**Issue by anasoid** [blueman-project/blueman#2097]
blueman: 2.3.5
BlueZ: 
Distribution: linux mint 21
Desktop environment: cinnamon 5.6

I have many neighbors with bluetooth device (phone, watch, TV) So i block all there device on blueman manager but i have a big list on it, there is no way to hide blocked devices?

**Comment by infirit** [comment]
There is currently no way.

I do wonder why you decided to block them as this is likely the reason they remain in the list.

**Comment by anasoid** [comment]
I connect regularity Bluetooth device, so if i doesn't block them some time by error if I click on the wrong line i will be connected to an unknown device

**Comment by infirit** [comment]
How about allowing you to hide any device without blocking it.

**Comment by anasoid** [comment]
thanks , yes it will be better

**Comment by DHager** [comment]
Yes please, from my apartment there are always neighbors' devices which I will *never* want to connect to.  (Currently 6 named, 17 unnamed.) 

So all those wrong (perhaps even dangerous) choices will clutter things up when I'm looking fora new device of my own, and if I "block" them for safety then they just show up even more.
